### PR TITLE
Add specific versions to dependencies

### PR DESCRIPTION
**Pull request by Shuhin**

### CODE REVIEW

File: src/main/resources/db/changelog/v1/data/csv/student_applications.csv [hunk 1]
```diff
@@ -16,6 +16,5 @@ id,applied_date,approved_by_id,name,date_of_birth,blood_group,birth_registration
 15,2021-01-08 06:00:01,2,Emilia Poundford,2005-04-27 06:00:01,B-,201499041029613,74-5189159,5514 Manley Trail,12601 Dawn Drive,Gamaliel Isakson,,548 272 9622,3
 16,2020-08-28 06:00:01,4,Letty Zannutti,2005-09-02 06:00:01,A+,6767327563346656,08-6077873,6 Rockefeller Alley,095 Walton Center,Gail Krink,,668 408 9708,2
 17,2020-05-10 06:00:01,3,Lilllie Flavelle,2014-08-08 06:00:01,A+,6.37538E+15,43-3649253,7263 Fulton Place,24718 Farmco Circle,Myrle Trafford,mtraffordg@cpanel.net,683 714 0021,2
-18,2021-01-23 06:00:01,5,Jud Deniskevich,2006-09-29 06:00:01,AB+,4017956045697659,70-4966997,912 Eliot Plaza,959 Bunker Hill Park,Zed Rubertelli,zrubertellih@google.fr,821 434 9172,1
```
Comment (ishmum123) [inline src/main/resources/db/changelog/v1/data/csv/student_applications.csv:19]: why is this necessary for specifying dependency versions?

Comment (Shuhin) [reply src/main/resources/db/changelog/v1/data/csv/student_applications.csv:19]: The id 5 is used in adminControllerTest ... And as approved_by_id is a foreign_key, without doing this the test shouldDeleteAdmin fails and overall the maven test shows failure.

**Comment by ishmum123** [comment]
https://github.com/ishmum123/studentlog/pull/24#discussion_r620842726

shouldn't that be in a separate PR in that case?

**Comment by Shuhin** [comment]
> [#24 (comment)](https://github.com/ishmum123/studentlog/pull/24#discussion_r620842726)
> 
> shouldn't that be in a
